activity-stream/simple-video-snippet.html

```diff
@@ -214,18 +214,26 @@ <h1>{{ scene1_title }}</h1>
       const video = document.getElementById('video');
 
       closeVideoButton.addEventListener('click', e => {
+        sendMetric('video-done');
+
         video.pause();
 
         scene2.classList.add('hidden');
         scene1.classList.remove('hidden');
       });
 
       showVideoButton.addEventListener('click', e => {
+        sendMetric('button-click');
+
         scene1.classList.add('hidden');
         scene2.classList.remove('hidden');
 
         video.play();
       });
+
+      video.addEventListener('ended', () => {
+        sendMetric('conversion-video-complete');
```

Comment:
We need an addition `conversion-video-play` metric when user clicks play

Reply:
I talked to Jean on Slack regarding this. The above handler (`showVideoButton`) starts playing the video, so the metric contained there (`button-click`) implies a video play. We agreed that adding `conversion-video-play` would be redundant.

Apologies for not documenting in the issue.